.github/workflows/codespell.yml

```
@@ -0,0 +1,25 @@
+# Codespell configuration is within .codespellrc
+---
+name: Codespell
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

```suggestion
  [push, pull_request]
```

---

we could do that, but likely would need to get to the same line as `on: `?

---

but also it would lead to double runs I think -- it might then run for the PR branch (e.g. "blah") and also then for the PR itself... adding noise etc.  So -- are you sure? what are you trying to achieve with widening the scope of when to run?

---

I would want tests to run when we make PRs from feature branches to our dev branch (we don't merge features directly to master) or between our feature branches.

---

so let's add `dev` there?
```suggestion
  push:
    branches: [master, dev]
  pull_request:
    branches: [master, dev]
```
so that pushes and PRs against `dev` are also tested?

---

That wouldn't include feature->feature PRs.

---

oh, I have missed the "or between our feature branches" part... ok -- your wish my act ... although I think this might be too much and you might want later to make it for pushes only to `master` and `dev` and all other PRs.